stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/mentions/MentionsListAdapter.kt

```diff
@@ -8,13 +8,17 @@ import androidx.recyclerview.widget.ListAdapter
 import androidx.recyclerview.widget.RecyclerView
 import com.getstream.sdk.chat.utils.DateFormatter
 import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.livedata.ChatDomain
 import io.getstream.chat.android.ui.databinding.StreamUiItemMentionListBinding
 import io.getstream.chat.android.ui.mentions.MentionsListAdapter.MessagePreviewViewHolder
 import io.getstream.chat.android.ui.mentions.MentionsListView.MentionSelectedListener
 import io.getstream.chat.android.ui.messagepreview.MessagePreviewView
+import io.getstream.chat.android.ui.utils.extensions.asMention
+import io.getstream.chat.android.ui.utils.extensions.context
 
 public class MentionsListAdapter(
-    context: Context
+    context: Context,
+    private val chatDomain: ChatDomain,
```

Comment:
I would pass only the required data: currentUser instead of ChatDomain

Reply:
Did this just in case the current user can somehow change while the adapter instance is alive (which is very unlikely, but still)